src/libstrongswan/utils/backtrace.c

```diff
@@ -408,6 +408,8 @@ static void print_sourceline(FILE *file, char *filename, void *ptr, void *base)
 void backtrace_init() {}
 void backtrace_deinit() {}
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-function"
```

Comment:
What is missing from the environment that causes this function to not be used?

Reply:
Obviously some `#ifdef` mess in there.

Not enough time to check / fix all StrongSwan issues.
Please keep comments relevant to PR scope which was to build for `aarch64`.

Reply:
This is very much relevant to this PR... If the apk you're shipping doesn't have support for printing backtraces, then we need to know about it.

Reply:
most likely `musl` libc implementation does not provide support for this, thus we won't have backtrace support.